### PR TITLE
Update M4 versions

### PR DESCRIPTION
**Pull request by BuddhiWathsala**
## Purpose
$subject

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,4 +1,4 @@
 org.gradle.caching=true
 group=org.ballerinalang
-version=0.6.1-SNAPSHOT
-ballerinaLangVersion=2.0.0-SNAPSHOT
+version=0.6.1-Preview1-M4
```
Comment (NipunaRanasinghe) [inline gradle.properties:3]: @BuddhiWathsala Any specific reason why we have to keep the release version as `0.6.1-Preview1-M4` instead of `0.6.1`?

Comment (BuddhiWathsala) [reply gradle.properties:3]: In GitHub public packages we can't change the published artifact after a release. So, if we we use `0.6.1` in this release, we cannot do further releases such as RC1 and even major release under this version.

Ref: https://help.github.com/en/packages/publishing-and-managing-packages/deleting-a-package

Comment (NipunaRanasinghe) [reply gradle.properties:3]: > In GitHub public packages we can't change the published artifact after a release. So, if we we use `0.6.1` in this release, we cannot do further releases such as RC1 and even major release under this version.
> 
> Ref: https://help.github.com/en/packages/publishing-and-managing-packages/deleting-a-package

@BuddhiWathsala In that case, I guess it'd better to just bump the release version (patch/minor) instead, as now the standard libs are having independent release cycles. @shafreenAnfar WDYT?

Comment (shafreenAnfar) [reply gradle.properties:3]: Yeah, now that socket module is detached, we don't have to tie it with ballerina-lang versions.

Comment (BuddhiWathsala) [reply gradle.properties:3]: @NipunaRanasinghe @shafreenAnfar @daneshk shall we go ahead with `0.6.1` version?

Comment (NipunaRanasinghe) [reply gradle.properties:3]: +1

Comment (daneshk) [reply gradle.properties:3]: Yes, let's go with 0.6.1 version. we can always bump to the next version if needed.

Comment (BuddhiWathsala) [reply gradle.properties:3]: +1 for the approach. Here to publish the GitHub packages without conflict, the better way to publish a release through a separate release branch. 

So, please refer to this release artifacts.

https://github.com/ballerina-platform/module-ballerina-socket/packages/237710

**Pull request closed without merging**